doc/source/installation/post-installation.rst

```diff
@@ -0,0 +1,177 @@
+.. _post-installation:
+
+===============================================================================
+Post-Installation
+===============================================================================
+
+After you have installed |Percona Server|, you may need to do the following:
+
+* Initialize the data directory. Some installation methods, such as ``apt`` or ``yum``, automatically initialize the data directory.
+* Update the ``root`` password. The data directory initialization may
+  update the password.
+* Start the server
+* Configure the server to start on startup.
+* Test access to the server
+* Enable time zone recognition.
+
+Initializing the Data Directory
+-------------------------------
+
+Some installation methods automatically initialize the data directory. If you have installed the server using source distribution or generic binary distribution files, the data directory is not initialized. You only need to run the initialization process after you have first installed the server. 
```

Comment:
this can be one sentence. `If you install ... from $method ... data dir is initialised by default, otherwise you need to do it by yourself.`

This provides information to user which method suits him better.

Reply:
Rewrote the paragraph.